src/listener/tcp_listener.rs

```diff
@@ -87,8 +91,9 @@ impl<State: Clone + Send + Sync + 'static> Listener<State> for TcpListener {
                     continue;
                 }
 
-                Ok(stream) => {
-                    handle_tcp(app.clone(), stream);
+                Ok(_stream) => {
+                    #[cfg(feature = "h1-server")]
+                    handle_tcp(_app.clone(), _stream);
```

Comment:
This is probably not the right solution. Any better ideas?

Reply:
I think there's no reason to include any of the Listener implementations unless h1-server is provided

Reply:
Addressed this in #678